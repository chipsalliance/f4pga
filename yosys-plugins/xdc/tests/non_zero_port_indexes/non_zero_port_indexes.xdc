set_property -dict { PACKAGE_PIN H5   IOSTANDARD LVCMOS33 } [get_ports { LED[2] }];
set_property -dict { PACKAGE_PIN J5   IOSTANDARD LVCMOS33 } [get_ports { LED[3] }];
set_property -dict { PACKAGE_PIN T9   IOSTANDARD LVCMOS33 } [get_ports { LED[4] }];
set_property -dict { PACKAGE_PIN T10   IOSTANDARD LVCMOS33 } [get_ports { LED[5] }];
