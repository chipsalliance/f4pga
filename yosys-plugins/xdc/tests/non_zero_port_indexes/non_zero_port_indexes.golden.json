{
	"$iopadmap$top.LED": {
		"IOSTANDARD": "LVCMOS33",
		"IO_LOC_PAIRS": "LED[2]:H5"
	},
	"$iopadmap$top.LED_1": {
		"IOSTANDARD": "LVCMOS33",
		"IO_LOC_PAIRS": "LED[3]:J5"
	},
	"$iopadmap$top.LED_2": {
		"IOSTANDARD": "LVCMOS33",
		"IO_LOC_PAIRS": "LED[4]:T9"
	},
	"$iopadmap$top.LED_3": {
		"IOSTANDARD": "LVCMOS33",
		"IO_LOC_PAIRS": "LED[5]:T10"
	}
}
